### DISCUSSION TITLE
Can we connect superset to hdfs without using hive.?

### DISCUSSION THREAD

**Discussion by pawar20396** [apache/superset#19949] [Q&A / Help]
I am new to big data. I just installed the apache superset. I want to access my Hadoop data from superset. Is there any option available? If yes then please specify how to do it?
Thank you for your support.

**Comment by zhaoyongjie** [comment]
@pawar20396 the HDFS is a filesystem, while Superset needs to access a database interface. You should expose a specific application from the Hadoop ecosystem rather than directly access a filesystem.

**Comment by pawar20396** [comment]
Do I need to use a hive for it?

**Comment by zhaoyongjie** [comment]
@pawar20396 may be Presto/Trino is better than Hive. Hive is for the batch jobs.

**Reply by pawar20396** [reply]
Hello, I connected superset using Spark SQL beeline ie jdbc:hive2://localhost:10000. I created two tables in my database. My connection is successfully connected. In superset, I can see database names. Superset shows database names multiple times even in the table name column so I am unable to select any table. Is there any solution for this?
